test/templates/pages/empty.njk

```diff
@@ -0,0 +1,26 @@
+{% extends "./layouts/page.njk" %}
```

Comment:
There is a special template set up for starter pages that remove some of the app-level containers not required for these "module" pages. See https://github.com/SAP/techne/blob/v2.0/test/templates/pages/tree.njk

```
{% extends "./layouts/module.njk" %}
```

The template includes some variables you can set to output the common page elements. The `is_landing_page` determines whether the back button is shown or not.

```
{% set page_title = "Tree" %}
{% set page_intro = "Page level help message goes here" %}
{% set is_landing_page = false %}
{% set show_toolbar = false %}
```

All you have to put in is the `page_content`. The rest is done for you.